examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientS2A.java

```diff
@@ -0,0 +1,118 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.helloworldtls;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import io.grpc.examples.helloworld.GreeterGrpc;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+import io.grpc.netty.GrpcSslContexts;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.net.ssl.SSLException;
+import javax.annotation.Nonnull;
+
+// TODO: change flags to use com.google.common.flags
+
+/**
+ * Establishes a TLS 1.3 connection with a greeter service.
+ *
+ * args:
+ * [ServerAddress] In the form of host:port
+ * [clientCertChainFilePath] File Path to Client Certificate
+ * [clientPrivateKeyFilePath] File Path to Client Key
+ * [trustCertCollectionFilePath] File Path to CA Certificate
+ */
+public class HelloWorldClientS2A {
+    private static final Logger logger = Logger.getLogger(HelloWorldClientS2A.class.getName());
+    private final ManagedChannel channel;
+    private GreeterGrpc.GreeterBlockingStub blockingStub;
+
+    private static SslContext buildSslContext(String clientCertChainFilePath,
+                                              String clientPrivateKeyFilePath,
+                                              String trustCertCollectionFilePath) throws SSLException {
+      checkNotNull(clientCertChainFilePath, "clientCertChainFilePath");
+      checkNotNull(clientPrivateKeyFilePath, "clientPrivateKeyFilePath");
+      checkNotNull(trustCertCollectionFilePath, " trustCertCollectionFilePath");
+      return GrpcSslContexts.forClient()
+            .trustManager(new File(trustCertCollectionFilePath))
+            .keyManager(new File(clientCertChainFilePath), new File(clientPrivateKeyFilePath))
+            .protocols("TLSv1.3")
+            .build();
+    }
+
+    /**
+     * Constructs client for accessing greeter server using the existing channel.
+     */
+    private HelloWorldClientS2A(ManagedChannel channel, GreeterGrpc.GreeterBlockingStub blockingStub) {
+        this.channel = channel;
+        this.blockingStub = blockingStub;
+    }
+
+    private void shutdown() throws InterruptedException {
+        channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
+    }
+
+    /**
+     * Say hello to server.
+     */
+    private void greet(String name) {
+        logger.info("Will try to greet " + name + " ...");
+        HelloRequest request = HelloRequest.newBuilder().setName(name).build();
+        HelloReply response;
+        try {
```

Comment:
Please omit this try/catch block and only have  the `response = blockingStub...` line. Doing it this way will mean we don't get a stack trace when an occur and, since this class will be used for testing the Go client libraries, we definitely want to always get a stack trace.